docs/modules/logstash.md

```diff
@@ -1,8 +1,11 @@
 # Logstash
 
 ### Purpose
+[Logstash](https://www.elastic.co/products/logstash) is an open-source tool for collecting, parsing, and storing logs for future use. Logstash will run alongside Kibana and Elasticsearch. 
```

Comment:
Can we link back to the Kibana and Elasticsearch modules? Also, can you make a guess as to the overall purpose of Logstash and why it will live on a dedicated server `log` that is connected to all other servers in the project, more so than just storing them for future use? This is definitely worth an open discussion to expand on the overall purpose of why we're looking at using this module.